merlin/spec/__init__.py

```diff
@@ -27,3 +27,20 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 ###############################################################################
+
+"""
+The `spec` package defines the structure, defaults, and functionality for working with
+Merlin specification files.
+
+Modules:
+    all_keys.py: Defines all the valid keys for each block in a Merlin specification file,
```

Comment:
no way to see these keys from here. I dont see a link to source

Reply:
The keys are just constants in the `all_keys.py` file. Mkdocstrings automatically provides the source code blocks but it doesn't do it for entire files (at least not to my knowledge).